bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,551 @@
+
+	<channel-type id="service">
+		<item-type>String</item-type>
+		<label>Service</label>
+		<description>Service Type</description>
+		<state readOnly="true">
+			<options>
+				<option value="Electric">Electric</option>
+				<option value="Hybrid">Hybrid</option>
+				<option value="Unknown">Unknown</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="preconditioning">
+		<item-type>String</item-type>
+		<label>Air Conditioning</label>
+		<description>Air conditioning status</description>
+		<state readOnly="true">
+			<options>
+				<option value="Enabled">Enabled</option>
+				<option value="Disabled">Disabled</option>
+				<option value="Finished">Finished</option>
+				<option value="Failure">Failure</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="preconditioningFailure">
+		<item-type>String</item-type>
+		<label>Air Conditioning Failure</label>
+		<description>Airr conditioning failure cause</description>
+		<state readOnly="true">
+			<options>
+				<option value="Defect">Defect</option>
+				<option value="DoorOpened">Door Opened</option>
+				<option value="LowBattery">Low Battery</option>
+				<option value="LowFuelLevel">Low Fuel Level</option>
+				<option value="TooManyUnusedProg">Too many times unused</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-group-type id="fuel">
+		<label>Fuel</label>
+		<description>Thermic Motor Fuel Status</description>
+		<category>Energy</category>
+		<channels>
+			<channel id="autonomy" typeId="autonomy"/>
+			<channel id="consumption" typeId="consumption"/>
+			<channel id="level" typeId="level"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="electric">
+		<label>Electric Energy</label>
+		<description>Electric energy</description>
+		<category>Electric motor energy status</category>
+		<channels>
+			<channel id="autonomy" typeId="autonomy">
+				<label>Electric Autonomy</label>
+				<description>Remaining distance</description>
+			</channel>
+			<channel id="level" typeId="level">
+				<label>Electricity Level</label>
+				<description>Electricity level</description>
+			</channel>
+			<channel id="residual" typeId="residual"/>
+			<channel id="batteryCapacity" typeId="capacity"/>
+			<channel id="batteryHealthCapacity" typeId="healthCapacity"/>
+			<channel id="batteryHealthResistance" typeId="healthResistance"/>
+			<channel id="chargingStatus" typeId="chargingStatus"/>
+			<channel id="chargingMode" typeId="chargingMode"/>
+			<channel id="chargingPlugged" typeId="chargingPlugged"/>
+			<channel id="chargingRate" typeId="chargingRate"/>
+			<channel id="chargingRemainingTime" typeId="chargingRemainingTime"/>
+			<channel id="chargingNextDelayedTime" typeId="chargingNextDelayedTime"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="level">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Fuel Level</label>
+		<description>Fuel level</description>
+		<category>Pressure</category>
```

Comment:
What's the relation to pressure? Same for below.

Reply:
I just liked the icon. Will remove.